guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -123,7 +125,7 @@ Marks a variable read-only and suitable for inlining. Generates warnings if it i
 mynamespace.DEFAULT_TIMEZONE&nbsp;=&nbsp;'GMT';
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-extends">@extends</h4>
+#### @extends
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-extends}` to preserve the custom id